nav2_bringup/launch/localization_launch.py

```diff
@@ -125,37 +125,40 @@ def generate_launch_description():
                 executable='lifecycle_manager',
                 name='lifecycle_manager_localization',
                 output='screen',
-                parameters=[{'use_sim_time': use_sim_time},
-                            {'autostart': autostart},
+                parameters=[{'autostart': autostart},
                             {'node_names': lifecycle_nodes}])
         ]
     )
 
-    load_composable_nodes = LoadComposableNodes(
+    load_composable_nodes = GroupAction(
         condition=IfCondition(use_composition),
```

Comment:
I would think that using the `condition` of LoadComposableNodes or GroupAction is the same